lab3/lab3/src/lab3/CreateFoodAdminServlet.java

```diff
@@ -10,32 +13,34 @@
 /**
  * Servlet implementation class CreateFoodAdminServlet
  */
-@WebServlet("/CreateFoodAdminServlet")
+@WebServlet("/admin/foods/create")
 public class CreateFoodAdminServlet extends HttpServlet {
-	private static final long serialVersionUID = 1L;
-       
-    /**
-     * @see HttpServlet#HttpServlet()
-     */
-    public CreateFoodAdminServlet() {
-        super();
-        // TODO Auto-generated constructor stub
-    }
-
-	/**
-	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
-	 */
+	
 	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-		// TODO Auto-generated method stub
-		response.getWriter().append("Served at: ").append(request.getContextPath());
+		
+		response.setContentType("text/html");
+		PrintWriter out = response.getWriter();
+		out.println("<form method=\"post\">");
+		out.println("<h1>Create New Food</h1>");
+		out.println("Food Name: <input name='name' type='text'/></br>");
+		out.println("Description: <textarea name='description'></textarea></br>");
+		out.println("Image URL: <input name='imageurl' type='text'/></br>");
+		out.println("Price: <input price='price' type='text'/></br>");
```

Comment:
`price="price"` is wrong, thus ending up with the adding food functionality isn't completed.